backend/migrations/20200704234350_create_userTimeRecord.js

```diff
@@ -0,0 +1,13 @@
+exports.up = function (knex) {
+    return knex.schema.createTable('userTimeRecords', (table) => {
+        table.increments();
+        table.integer('userId').unsigned().references('users.id').notNullable();
+        table.integer('projectId').unsigned().references('projects.id').notNullable();
```

Comment:
miért kell letárolni a projectId-t, ha itt van a milestoneId, amiből megmondható a projectId is?

Reply:
kiszedtem. get-nél visszajön, többinél nem kéri be